gradle/libs.versions.toml

```diff
@@ -11,7 +11,7 @@ ksp = { id = "com.google.devtools.ksp", version.ref = "kotlin-ksp" }
 project = "1.0"
 
 java = "1.8"
-agp = "8.2.2"
+agp = "8.3.0"
```

Comment:
<details><summary>Lint failures</summary>

```
Picked up JAVA_TOOL_OPTIONS: -Xms32M -Xmx256M -XX:+UseG1GC -D-XX:+PrintCommandLineFlags
Configuration on demand is an incubating feature.
Reusing configuration cache.

> Task :annotations:verifyReleaseResources
WARNING: The specified Android SDK Build Tools version (33.0.2) is ignored, as it is below the minimum supported version (34.0.0) for Android Gradle Plugin 8.3.0.
Android SDK Build Tools 34.0.0 will be used.
To suppress this warning, remove "buildToolsVersion '33.0.2'" from your build.gradle file, as each version of the Android Gradle Plugin now has a default version of the build tools.

> Task :glide4:kspReleaseKotlin
w: [ksp] LibraryGlideModules [] are included more than once, keeping only one!

> Task :glide4:kspDebugKotlin
w: [ksp] LibraryGlideModules [] are included more than once, keeping only one!

> Task :defs:testDebugUnitTest
Picked up JAVA_TOOL_OPTIONS: -Xms32M -Xmx256M -XX:+UseG1GC -D-XX:+PrintCommandLineFlags

Java HotSpot(TM) 64-Bit Server VM warning: Sharing is only supported for boot loader classes because bootstrap classpath has been appended
Java HotSpot(TM) 64-Bit Server VM warning: Sharing is only supported for boot loader classes because bootstrap classpath has been appended

> Task :defs:testReleaseUnitTest
Picked up JAVA_TOOL_OPTIONS: -Xms32M -Xmx256M -XX:+UseG1GC -D-XX:+PrintCommandLineFlags

> Task :basics:testReleaseUnitTest
Picked up JAVA_TOOL_OPTIONS: -Xms32M -Xmx256M -XX:+UseG1GC -D-XX:+PrintCommandLineFlags

> Task :basics:testDebugUnitTest
Picked up JAVA_TOOL_OPTIONS: -Xms32M -Xmx256M -XX:+UseG1GC -D-XX:+PrintCommandLineFlags

> Task :monolith:testReleaseUnitTest
Picked up JAVA_TOOL_OPTIONS: -Xms32M -Xmx256M -XX:+UseG1GC -D-XX:+PrintCommandLineFlags

Java HotSpot(TM) 64-Bit Server VM warning: Sharing is only supported for boot loader classes because bootstrap classpath has been appended

> Task :monolith:testDebugUnitTest
Picked up JAVA_TOOL_OPTIONS: -Xms32M -Xmx256M -XX:+UseG1GC -D-XX:+PrintCommandLineFlags

> Task :espresso:testReleaseUnitTest
Picked up JAVA_TOOL_OPTIONS: -Xms32M -Xmx256M -XX:+UseG1GC -D-XX:+PrintCommandLineFlags

> Task :orbit:testDebugUnitTest
Picked up JAVA_TOOL_OPTIONS: -Xms32M -Xmx256M -XX:+UseG1GC -D-XX:+PrintCommandLineFlags

> Task :orbit:testReleaseUnitTest
Picked up JAVA_TOOL_OPTIONS: -Xms32M -Xmx256M -XX:+UseG1GC -D-XX:+PrintCommandLineFlags

Java HotSpot(TM) 64-Bit Server VM warning: Sharing is only supported for boot loader classes because bootstrap classpath has been appended

> Task :espresso:testDebugUnitTest
Picked up JAVA_TOOL_OPTIONS: -Xms32M -Xmx256M -XX:+UseG1GC -D-XX:+PrintCommandLineFlags

> Task :utils:collect:test
Picked up JAVA_TOOL_OPTIONS: -Xms32M -Xmx256M -XX:+UseG1GC -D-XX:+PrintCommandLineFlags

> Task :lib:junit4:test
Picked up JAVA_TOOL_OPTIONS: -Xms32M -Xmx256M -XX:+UseG1GC -D-XX:+PrintCommandLineFlags
Java HotSpot(TM) 64-Bit Server VM warning:
> Task :lib:mockito:test
Picked up JAVA_TOOL_OPTIONS: -Xms32M -Xmx256M -XX:+UseG1GC -D-XX:+PrintCommandLineFlags

Sharing is only supported for boot loader classes because bootstrap classpath has been appended

> Task :stringers:testDebugUnitTest
Picked up JAVA_TOOL_OPTIONS: -Xms32M -Xmx256M -XX:+UseG1GC -D-XX:+PrintCommandLineFlags

> Task :lib:java:test
Picked up JAVA_TOOL_OPTIONS: -Xms32M -Xmx256M -XX:+UseG1GC -D-XX:+PrintCommandLineFlags

> Task :lib:general:test
Picked up JAVA_TOOL_OPTIONS: -Xms32M -Xmx256M -XX:+UseG1GC -D-XX:+PrintCommandLineFlags

Java HotSpot(TM) 64-Bit Server VM warning: Sharing is only supported for boot loader classes because bootstrap classpath has been appended

> Task :lib:hamcrest:test
Picked up JAVA_TOOL_OPTIONS: -Xms32M -Xmx256M -XX:+UseG1GC -D-XX:+PrintCommandLineFlags

Java HotSpot(TM) 64-Bit Server VM warning: Sharing is only supported for boot loader classes because bootstrap classpath has been appended

> Task :utils:stringer:test
Picked up JAVA_TOOL_OPTIONS: -Xms32M -Xmx256M -XX:+UseG1GC -D-XX:+PrintCommandLineFlags

Java HotSpot(TM) 64-Bit Server VM warning: Sharing is only supported for boot loader classes because bootstrap classpath has been appended

> Task :stringers:testReleaseUnitTest
Picked up JAVA_TOOL_OPTIONS: -Xms32M -Xmx256M -XX:+UseG1GC -D-XX:+PrintCommandLineFlags

> Task :utils:test:test
Picked up JAVA_TOOL_OPTIONS: -Xms32M -Xmx256M -XX:+UseG1GC -D-XX:+PrintCommandLineFlags

> Task :basics:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\basics\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :annotations:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\annotations\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :about:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\monolith\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :color_picker:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\color_picker\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :defs:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\defs\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :cpsuite:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\cpsuite\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :capture_image:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\glide4\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :espresso:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\monolith\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :espresso_actors:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\espresso_actors\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :about-test_helpers:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\about\test_helpers\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :capture_image-test_helpers:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\capture_image\test_helpers\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :espresso_glide3:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\monolith\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :glide4:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\glide4\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :logging:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\logging\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :espresso_glide4:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\monolith\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :monolith:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\monolith\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :glide3:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\glide3\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :permissions:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\permissions\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :settings:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\settings\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :stringers:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\stringers\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :uiautomator:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\uiautomator\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :slf4j:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\slf4j\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :widgets:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\widgets\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :orbit:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\orbit\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :internal:test:android_instrumentation:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\internal\test\android_instrumentation\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

> Task :internal:test:android_unit:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\internal\test\android_unit\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.

FAILURE: Build completed with 26 failures.

1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':basics:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':annotations:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

3: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':about:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

4: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':color_picker:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

5: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':defs:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

6: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':cpsuite:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

7: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':capture_image:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

8: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':espresso:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

9: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':espresso_actors:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

10: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':about-test_helpers:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

11: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':capture_image-test_helpers:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

12: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':espresso_glide3:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

13: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':glide4:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

14: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':logging:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

15: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':espresso_glide4:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

16: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':monolith:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

17: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':glide3:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

18: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':permissions:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

19: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':settings:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

20: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':stringers:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

21: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':uiautomator:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

22: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':slf4j:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

23: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':widgets:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

24: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':orbit:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

25: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':internal:test:android_instrumentation:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

26: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':internal:test:android_unit:lintReportDebug'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Internal error: Unexpected lint invalid arguments

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

BUILD FAILED in 1m 18s
5289 actionable tasks: 5289 executed
Configuration cache entry reused.
```

</details>

Reply:
tasks are not running
```
> Task :basics:generateDebugLintReportModel UP-TO-DATE
> Task :basics:mergeDebugConsumerProguardFiles UP-TO-DATE
> Task :basics:lintAnalyzeDebug UP-TO-DATE
> Task :basics:bundleDebugLocalLintAar UP-TO-DATE
> Task :basics:generateDebugAndroidTestLintModel SKIPPED
> Task :basics:lintAnalyzeDebugAndroidTest SKIPPED
> Task :basics:generateDebugUnitTestLintModel UP-TO-DATE
> Task :basics:lintAnalyzeDebugUnitTest UP-TO-DATE

> Task :basics:lintReportDebug FAILED
Lint model P:\projects\workspace\net.twisterrob.travel\libs\twister-lib-android\basics\build\intermediates\android_test_lint_model\debug\generateDebugAndroidTestLintModel does not exist.
```

Reply:
`--info`

```
> Task :defs:generateDebugAndroidTestLintModel SKIPPED
Skipping task ':defs:generateDebugAndroidTestLintModel' as task onlyIf 'Task is enabled' is false.

> Task :defs:lintAnalyzeDebugAndroidTest SKIPPED
Skipping task ':defs:lintAnalyzeDebugAndroidTest' as task onlyIf 'Task is enabled' is false.
```

Reply:
https://github.com/TWiStErRob/net.twisterrob.libraries/pull/127/commits/b86229d032391ea9afe59ed7b82cfedbbe2a14e3